cvat/apps/engine/views.py

```diff
@@ -242,24 +258,37 @@ def annotations(self, request, pk):
                 except (AttributeError, IntegrityError) as e:
                     return Response(data=str(e), status=status.HTTP_400_BAD_REQUEST)
                 return Response(data)
+        elif request.method == 'POST':
```

Comment:
In the past we lived without POST method. Why do we need it now? PUT - replace, PATCH - modify. What does POST method in the case?